badger_voter_sdk/collectors/data_processors.py

```diff
@@ -0,0 +1,37 @@
+from collections import defaultdict
+from decimal import Decimal
+from typing import Dict
+from typing import Optional
+
+from web3 import Web3
+
+
+def extract_voting_power_per_pool(
+        voters: Dict[str, Dict], scores: Dict
+) -> Optional[Dict[str, Decimal]]:
+    """
+    Extract voting power for each pool, combining two datasets:
+    1. voters - has weighed votes per wallet
+    2. scores - has data about vlTOKEN voted by wallet
+
+    Returns voting choices calculated in a form {"1": Decimal(123123.04), ...}
+    """
+    if not voters or not scores:
+        return
```

Comment:
should log here or raise?

Reply:
This is library. I don't think it's good to generate logs by library